### ISSUE TITLE
Change the default form (related content) name so it doesn't reference driver

### ISSUE THREAD

**Issue by jeancochrane** [azavea/ashlar-schema-editor#15]
The default name for a form (related content) is `driver<EntityName>Details`:

![driverdetails](https://user-images.githubusercontent.com/14170650/44353605-7ec88700-a475-11e8-97d1-b0be90991a61.png)

This doesn't make much sense now that the schema editor is its own separate app.

**Comment by ddohler** [comment]
Indeed! One note about this that is potentially important: we sometimes have had schemas that were entirely in non-ASCII characters, and I believe the motivation for the `driver` prefix was that we encountered a bug in one of our libraries that we were able to fix by prefixing all form names with `driver` under the hood. I don't remember the exact details but I can probably dig them up if necessary.

**Comment by jeancochrane** [comment]
Great bit of history there! I'm guessing that the solution had to do with processing it as a string and not with the specific string `driver` itself, right? If so, one way that we could superstitiously try to avoid any encoding bugs would be to change the prefix to `default` so that e.g. `driverCatDetails` becomes `defaultCatDetails`. Still might be slightly confusing to a new user, but perhaps easier to grok than `driver`.

**Comment by flibbertigibbet** [comment]
That's right, @ddohler. The generated Java class names used for the Android implementation needed to follow naming rules requiring beginning with an ASCII character. Changing the prefix should be sufficient.
